cmd/cmd_flags.go

```diff
@@ -90,6 +98,47 @@ func getWebhookFlagOrArg(c *cli.Context) string {
 	return requireFlagOrArg(c, "webhook", "Use `webhookdb webhook list` to see available webhooks.")
 }
 
+func syncTargetFlag() *cli.StringFlag {
```

Comment:
Only common flags should go here (the `webhookFlag` here is a mistake). Move these into `cmd_synctarget`.